pkg/daemon/controller/controller.go

```diff
@@ -499,6 +504,21 @@ func (sc *ServiceController) resolvePowershellVariables(svc servicescm.Service)
 	return vars, nil
 }
 
+func (sc *ServiceController) resolveVaraiblesInPath(script servicescm.PowershellPreScript) (servicescm.PowershellPreScript, error) {
```

Comment:
Please add a comment documenting the function and consider adding a unit test in `controller_test.go` for the function as well, especially to cover the string replacement.

Reply:
Would this potential unit test test anything that resolvePowershellVariables is not already testing?